src/gearbox/main.py

```diff
@@ -72,11 +80,6 @@ async def validation_exception_handler(request:Request, exc:ValueError):
         content = {'status_code': 10422,'message': 'PYDANTIC ValidationError' + exc_str , 'data':None}
         return JSONResponse(content=content, status_code = status.HTTP_422_UNPROCESSABLE_ENTITY)
 
-#    @app.on_event("shutdown")
-#    async def shutdown_event():
-#        logger.info("Closing async client.")
-#        await app.async_client.aclose()
```

Comment:
unrelated to this ticket. Should we move the shutdown logic to this new event handler in fastapi? things like closing DB connection and such?